src/Breadcrumbs/Breadcrumbs.types.ts

```diff
@@ -0,0 +1,9 @@
+import { BreadcrumbsProps as MuiBreadcrumbsProps } from "@mui/material";
+import { PropsWithChildren } from "react";
+
+/**
+ * Breadcrumbs component props
+ */
+export type BreadcrumbsProps = PropsWithChildren<
+  Omit<MuiBreadcrumbsProps, "sx">
```

Comment:
If we're excluding `sx`, it feels like we should probably also exclude `style`, `className`, and `classes`.